tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -1240,6 +1241,65 @@ public function userTriesToDisableShareSyncOfResourceUsingTheGraphApi(string $us
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @param string $user
+	 * @param string $resource
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function isShareSynced(string $user, string $resource): bool {
+		$resource = \trim($resource, '/');
+		$response = GraphHelper::getSharesSharedWithMe(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$user,
+			$this->featureContext->getPasswordForUser($user)
+		);
+
+		$shares = $this->featureContext->getJsonDecodedResponse($response)["value"];
+		$syncStatus = false;
+		foreach ($shares as $share) {
+			if ($share["name"] === $resource) {
+				$syncStatus = $share["@client.synchronize"];
+				break;
+			}
+		}
+		return $syncStatus === true;
```

Comment:
This seems not necessary. If `$syncStatus` is a boolean, then it can be returned as-is.

If `$syncStatus` is some other type (for example, the integer `1` or `0`) then this will always return false. Maybe assert that `$syncStatus` is `boolean` so that things will fail early if the json decoded response does not have the expected type.